extras/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ function(rwkv_add_extra source)
 	get_filename_component(EXTRA_TARGET ${source} NAME_WE)
 	add_executable(rwkv_${EXTRA_TARGET} ${source})
 	target_link_libraries(rwkv_${EXTRA_TARGET} PRIVATE ggml rwkv)
+	if (RWKV_STATIC)
+		get_target_property(target_LINK_OPTIONS rwkv_${EXTRA_TARGET} LINK_OPTIONS)
+    	list(REMOVE_ITEM target_LINK_OPTIONS "-static")
+    	    set_target_properties(rwkv_${EXTRA_TARGET} PROPERTIES LINK_OPTIONS "${target_LINK_OPTIONS}")
```

Comment:
use tabs for indentation or convert the whole file to use spaces

Reply:
ok, i will do. It comes somehow wrong over sugestions

Reply:
it's fixed now

Reply:
@LoganDark @saharNooby no idea why, extras use tabs, and tests use spaces :D

Reply:
because I used tabs when contributing extras, it was an accident